src/main/groovy/JsJobBuilder.groovy

```diff
@@ -37,12 +49,14 @@ class JsJobBuilder {
                         '''
                 )
             }
+
             publishers {
                 archiveArtifacts artifacts
-                if (emails) {
-                    mailer emails.join(' ')
-                }
             }
+
         }
+
+        baseJob
```

Comment:
@Ooblioob You are wrong. What is returned is the last call, which in this case would be the with() function, which is void.